docker-entrypoint.sh

```diff
@@ -35,12 +35,27 @@ drop_privs_cmd() {
     fi
 }
 
+copy_plugins_if_required() {
+  echo "Enabling required built-in plugins"
+  for TARGET_PLUGIN in $(echo "$ENABLE_BUILT_IN_PLUGINS" | grep -o -e "[^;]*"); do
+    echo "Moving $TARGET_PLUGIN to plugin directory"
+    PLUGIN_NAME=${TARGET_PLUGIN%.jar}
```

Comment:
```suggestion
    plugin_name="${TARGET_PLUGIN%.jar}"
```

Lowercase to distinguish from globals and quoting assignment.